.github/workflows/template-check-connect-version-match.yml

```diff
@@ -0,0 +1,37 @@
+name: Check connect version match with branch
```

Comment:
This should probably be called `[Template] ...`

Reply:
Also i would probably convert this to action as this is something that is quite static  and wont be changing that often (but its super low priority)

Reply:
Yes, I made new action `.github/actions/check-connect-version-match/action.yml`